cmd/pipelines-as-code-controller/main.go

```diff
@@ -28,5 +32,30 @@ func main() {
 		log.Fatal("failed to init kinit client : ", err)
 	}
 
+	loggerConfiguratorOpt := evadapter.WithLoggerConfiguratorConfigMapName(logging.ConfigMapName())
+	loggerConfigurator := evadapter.NewLoggerConfiguratorFromConfigMap(PACControllerLogKey, loggerConfiguratorOpt)
+	copt := evadapter.WithLoggerConfigurator(loggerConfigurator)
+	// put logger configurator to ctx
+	ctx = evadapter.WithConfiguratorOptions(ctx, []evadapter.ConfiguratorOption{copt})
+	// set up kubernetes interface to retrieve configmap with log configuration
+	ctx = context.WithValue(ctx, client.Key{}, run.Clients.Kube)
+
+	ns, err := getPacNamespace()
+	if err != nil {
+		log.Fatal("failed to get current pac namespace : ", err)
+	}
+	ctx = evadapter.WithNamespace(ctx, ns)
+	ctx = evadapter.WithConfigWatcherEnabled(ctx)
+
 	evadapter.MainWithContext(ctx, PACControllerLogKey, adapter.NewEnvConfig, adapter.New(run, kinteract))
 }
+
+func getPacNamespace() (string, error) {
+	// retrieves the current pac namespace
+	nsBytes, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+	if err != nil {
+		return "", err
+	}
+
+	return string(nsBytes), nil
+}
```

Comment:
Can be removed completely

Reply:
the thing with having detection in cluster is that it would fail when runing controller/watcher under a debugger from a dev workstation, so rather use the env variables.

if we realy want to use that function we would need to refactor code to only use this (+fallback to env variable if file not there)